yt/geometry/oct_container.pyx

```diff
@@ -185,6 +185,16 @@ cdef class OctreeContainer:
     cdef np.int64_t get_domain_offset(self, int domain_id):
         return 0
 
+    def check_root_mesh(self):
+        cdef count = 0
+        for i in range(self.nn[0]):
+            for j in range(self.nn[1]):
+                for k in range(self.nn[2]):
+                    if self.root_mesh[i][j][k] == NULL:
+                        print("Missing ", i, j, k)
+                        count += 1
+        print("Missing total of %s out of %s" % (count, self.nn[0] * self.nn[1] * self.nn[2]))
```

Comment:
doesn't look like you're using this method anywhere, is it debugging code ? I'd suggesting dropping it before we can merge this

Reply:
I don't particularly want to drop this, as it would just need to be reimplemented the next time we need to debug something.  I am OK with making it semi-private.